### DISCUSSION TITLE
Mariner

### DISCUSSION THREAD

**Discussion by drashna** [rkolbi/My_Mars2Pro_Accessories#3] [General]
Do you have the USB data cable connected, *and* the 5V on the motherboard's header connected?   Form the images, it appears so.

**Comment by rkolbi** [comment]
Yes - both are connected as they are both coming from the same supply source. The rPiW has low power requirements and I have had no issues powering it from the printer's controller board.

*If you decide to power the rPi from an external power supply, the USB cable should have the +5 wire cut to avoid back feeding power to the printer control board.
